### ISSUE TITLE
2.0.133: issue with `ManagerDevices.getDevices()`

### ISSUE THREAD

**Issue by robertklep** [athombv/node-athom-api#2]
When using this code:
```
const devices = await homeyApi.devices.getDevices()
```

I get this error:
```
TypeError: Expected capabilities to be an Array, got object instead
    at Function.value (/Users/robert/Devel/Homey/homey-web-api-example/node_modules/athom-api/dist/index.js:1:656248)
    at /Users/robert/Devel/Homey/homey-web-api-example/node_modules/athom-api/dist/index.js:1:640579
    at Array.forEach (<anonymous>)
    at t.<anonymous> (/Users/robert/Devel/Homey/homey-web-api-example/node_modules/athom-api/dist/index.js:1:640510)
    at a (/Users/robert/Devel/Homey/homey-web-api-example/node_modules/athom-api/dist/index.js:1:323564)
    at Generator._invoke (/Users/robert/Devel/Homey/homey-web-api-example/node_modules/athom-api/dist/index.js:1:324606)
    at Generator.e.(anonymous function) [as next] (/Users/robert/Devel/Homey/homey-web-api-example/node_modules/athom-api/dist/index.js:1:323743)
    at n (/Users/robert/Devel/Homey/homey-web-api-example/node_modules/athom-api/dist/index.js:1:7941)
    at /Users/robert/Devel/Homey/homey-web-api-example/node_modules/athom-api/dist/index.js:1:8037
    at <anonymous>
```

It works in 2.0.115 (haven't tried other versions).

**Comment by Lokaltog** [comment]
Tested this with a newer version of the API because device.refreshCapabilities() is broken in 2.0.115. Version 2.0.131 is the latest working version, this bug seems to be introduced in 2.0.132.

**Comment by jeroenvollenbrock** [comment]
This should be fixed in 2.0.134

**Comment by Lokaltog** [comment]
The issue seems to be resolved in 2.0.135-2.0.141 but appears to be re-introduced from version 2.0.142 until the latest version.

**Comment by jeroenvollenbrock** [comment]
Fixed in 2.0.148, this is also going to be the last Homey v1.0-only release. athom-api@2.1.138 has been promoted to the beta channel with the Homey v2 compatible APIs. Some v1 backports for critical functions have been added and the docs have been updated to reflect this. Depending on your use-case you may want to use this version instead, so you're prepared for the Upcoming Homey v2.0.0 release.
